test/expected/build/library.j/dest/resources/library/j/ES6.features.2.js

```diff
@@ -21,14 +21,13 @@ window.someRandomModule ||
 		 * @version ${version}
 		 *
 		 * @constructor
-		 * @extends library.j.a
+		 * @extends library.j.Control
```

Comment:
This doesn't exist. I'd rather suggest to just import and extend from sap.ui.core.Control.

Reply:
Done